lib/postgrest.dart

```diff
@@ -32,6 +33,14 @@ class PostgrestClient {
   /// Perform a stored procedure call.
   PostgrestBuilder rpc(String fn, Map params) {
     final url = '${this.url}/rpc/$fn';
-    return PostgrestQueryBuilder(url, headers: headers, schema: schema).rpc(params);
+    return PostgrestQueryBuilder(url, headers: headers, schema: schema)
+        .rpc(params);
   }
 }
+
+/// Returns count as part of the response when specified.
+enum CountOption {
```

Comment:
We should create a new file `count_option.dart` under `lib/src` to define this enum.

Reply:
Sounds good! I was very unsure where to place this code. Thank you for your advice.